src/containers/course-section/resources-list/ResourcesList.tsx

```diff
@@ -8,12 +8,18 @@ import SortableWrapper from '~/containers/sortable-wrapper/SortableWrapper'
 import ResourceItem from '~/containers/course-section/resource-item/ResourceItem'
 import { styles } from '~/containers/course-section/resources-list/ResourcesList.styles'
 
-import { CourseResource, ResourceAvailability } from '~/types'
+import {
+  CourseResource,
+  ResourceAvailability,
+  ResourceAvailabilityStatusEnum,
+  Resource
+} from '~/types'
 
 import useDroppable from '~/hooks/use-droppable'
 import useDndSensor from '~/hooks/use-dnd-sensor'
 
 interface ResourcesListProps {
+  cooperationData?: Resource[]
   items: CourseResource[]
```

Comment:
Why do we need to pass `items` together with `cooperationData` as props, as I understand from the code we can always get `items` from `cooperationData`

Reply:
Fixed! Thank you for this suggestion